Gemfile

```diff
@@ -82,6 +82,7 @@ group :check do
     gem 'rbs', '~> 3.2.0', require: false
     gem 'steep', '~> 1.6.0', require: false
   end
+  gem 'standard'
```

Comment:
```
gem 'standard', require: false
```

(For some reason I can't make a suggestion).

Reply:
Thanks for catching it! Fixed in 252713bf1f7a21015c15db0ee37ba095a336697e